### DISCUSSION TITLE
Understanding the folder structure

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#1253] [General]
I see the scripts moved to 
~/.local/share/bin/

I also have 
~/.config/hypr obviously and 
~/.config/hyde (as I am on the "dev" branche.
~/.config/hyprdots
~/.config/hyprdots-ctl

~/.icons
~/.themes 
~/Hyprdots
(which I would rather not have in my ~/ 

Are all this folder necessary and/or can some be re(moved) to ~/.config/

**Comment by RookiexCookie** [comment]
speaking of the folder structure , in hyde branch why is there is Wall dcol , Wall Way in wallbash directory 
whats the difference

**Reply by kRHYME7** [reply]
dcol (dominant color)

Wall dcol is for dcol that have theme counterpart. 

Wall Ways will be always in wallbash mode because there is no per theme .dcol file for theme

**Comment by kRHYME7** [comment]
@se7en-x230  
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/52a99c1c-abda-46ab-9c63-5f6751d19e26)

Mark files are the once needed on the next update. 


For Hyde-cli  I'm trying to achieve this workflow

``` Hyde update ``` >>> Pull changes to the Clone Repository

```Hyde upgrade ``` >>>  works like install.sh  but it will ask you what files to populate

**Reply by se7en-x230** [reply]
so git pull pulls down the updates to ~/Hyprdots or ~/.config/hyprdots or are they not needed once installed?

**Comment by se7en-x230** [comment]
what I do in the moment is I start a custom script to pull the latest updates and have a ~/.config/hyp7 dir to copy my custom settings back. 

![image](https://github.com/prasanthrangan/hyprdots/assets/69777713/1ea47cc1-fc8f-4870-97bf-c5f05ba29d32)
